source/includes/connect/mongoclient2.c

```diff
@@ -0,0 +1,48 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main(void) {
+
+  //start-connect-to-atlas
+  mongoc_client_t *client;
+  mongoc_server_api_t *api = NULL;
+  bson_t *command, reply;
+  bson_error_t error;
+  char *uri;
+  bool retval;
+  bool ok = true;
+
+  // Intialize the MongoDB C Driver
+  mongoc_init();
+
+  // Create a new client and connect to the server
+  uri = "mongodb+srv://<db_username>:<db_password>@<hostname/port>/?<options>";
+  client = mongoc_client_new (uri);
+
+  // Set the version of the Stable API on the client
+  api = mongoc_server_api_new(MONGOC_SERVER_API_V1);
+  ok = mongoc_client_set_server_api(client, api, &error);
```

Comment:
```suggestion
  {
     mongoc_server_api_t *api = mongoc_server_api_new (MONGOC_SERVER_API_V1);
     if (!mongoc_client_set_server_api (client, api, &error)) { ... }
     mongoc_server_api_destroy (api);
  }
```
Reduce scope of variables when able.